client/src/components/User/DiskUsage/Visualizations/util.ts

```diff
@@ -0,0 +1,119 @@
+import { onMounted, ref } from "vue";
+
+import { useConfirmDialog } from "@/composables/confirmDialog";
+import { useToast } from "@/composables/toast";
+import localize from "@/utils/localization";
+
+import type { DataValuePoint } from "./Charts";
+import { bytesLabelFormatter, bytesValueFormatter } from "./Charts/formatters";
+import { type ItemSizeSummary, purgeDatasetById, undeleteDatasetById } from "./service";
+
+interface DataLoader {
+    (): Promise<undefined>;
```

Comment:
It means the same, but for me it seems a little bit more standard. Feel free to ignore.
```suggestion
    (): Promise<void>;
```

Reply:
Done!